src/tabs/tabs.jsx

```diff
@@ -19,6 +19,7 @@ let Tabs = React.createClass({
     tabWidth: React.PropTypes.number,
     tabItemContainerStyle: React.PropTypes.object,
     contentContainerStyle: React.PropTypes.object,
+    inkBarStyles: React.PropTypes.object,
```

Comment:
Please rename this to `inkBarStyle` to match others. Also, please add to docs site.